transit_odp/timetables/tests/test_extract_metadata.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
Need to add test cases that verifies expected functionality after transform and load. `test_etl*` from pipelines app can be referred